src/esm_runscripts/__init__.py

```diff
@@ -14,6 +14,7 @@
 from .last_minute import *
 from .dataprocess import *
 from .filelists import *
+from .filedicts import *
```

Comment:
I do not understand why this is needed here at the moment. In any case, wildcard "import everything" from a module is not a concept I would promote in the new functionality; I would prefer explicit and verbose imports, even if it means more typing.

Reply:
Actually, we don't even need this I guess. `from esm_runscripts import <module>` should do the trick. Maybe we can have a blank __init__ file like many modern projects. Currently, `__init__.py` is no longer needed to make a directory module.